source/scripts/model/Game.js

```diff
@@ -27,6 +27,12 @@ export default class Game {
                     MONSTERS.RED_ORC,
                     MONSTERS.BLUE_ORC,
                     MONSTERS.GREEN_ORC,
+                    MONSTERS.WHITE_TROLL,
+                    MONSTERS.RED_BAT,
+                    MONSTERS.BLUE_BAT,
+                    MONSTERS.GREEN_BAT,
+                    MONSTERS.FAST_BAT,
+                    MONSTERS.STONE_BAT,
```

Comment:
As a note, I appreciate you ending this line with a comma, so when the next person changes this file to add another monster, they don't "edit" your last line, which makes the git diff look confusing.